views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss

```diff
@@ -68,9 +73,10 @@ $modalShadowColor: #666666;
       }
 
       @include rtl {
-        float: right;
-        margin-right: 0;
-        margin-left: 10px;
+        margin-right: 0.7em;
+        margin-left: 0px;
+        flex-wrap: nowrap;
+        flex-direction: row-reverse;
```

Comment:
This code seems repeated. I suggest setting the margins outside of the @includes so it will be applied to both cases, and you only have the code once.
Does the `flex-wrap` affect the item? I can't see it.